src/test/integration/api/wiremock/mappings/uuidMapping.json

```diff
@@ -0,0 +1,18 @@
+{
+  "request": {
+    "method": "GET",
+    "url": "/uuid?email=dan.cotton@capgemini.com",
```

Comment:
see above comment! this is for server - response needs to differentiate between this and one for client. Origin may be a good way of doing this!